.github/container/manifest.yaml

```diff
@@ -1,32 +1,14 @@
-jax:
```

Comment:
Correct me if I'm wrong, but you suggest to use default values for jax `ref#commit` from Dockerfile rather than from manifest file?
What is the scenario for NGC release? Update Dockerfiles accordingly?

Reply:
Definitely yes for the normal CI: by default we take the Dockerfile defaults, which is basically "HEAD".

When preparing releases, one option (noted here: https://github.com/NVIDIA/JAX-Toolbox/pull/782#discussion_r1601300132) is to commit a manifest file to the release branch and use that. Your suggestion of changing the default values in the Docker files in the release branches would also work, I think, and would require less machinery (to translate manifest --> URLREF build arg) but might be a little more tedious to seed at the beginning of the release process.

cc: @terrykong @ko3n1g